drivers/SmartThings/matter-sensor/src/init.lua

```diff
@@ -157,8 +274,10 @@ local function do_configure(driver, device)
     profile_name = profile_name .. "-leak"
   end
 
-  if supports_battery_percentage_remaining(device) then
+  if device:get_field(SUPPORT_BATTERY_PERCENTAGE) then
```

Comment:
I believe there is a confluence page somewhere that indicates the expected order of capabilities in a profile. I cannot seem to find it, but I believe @hcarter-775 might have updated it somewhat recently.

Anyway, in a similar vein, we should document the expected order of our profile name tags so that when we are creating new profiles, we can ensure that their name are internally consistent. For example, this function essentially outlines the order I am talking about (motion is first, contact is second...preferences are last). This way it is clear the order of precedence when using all these combinations of flags to create the profile name.